.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+    * Question: Prefix your issue title with [QUESTION]
+-->
+
+<!--
+    This section is only for Bug and Question issues
+-->
+* Sponge Implementation: <!-- SpongeForge or SpongeVanilla -->
+* Sponge build version: <!-- Find the version using /sponge version -->
```

Comment:
I don't think an average server owner will actually know this. @hsyyid - I think we should make a command that outputs version information that a user can copy that includes Sponge information, that is, if we can. Then, we ask them to copy/paste this information into the issue.